tests/modules/asset_groups/test_models.py

```diff
@@ -45,25 +45,26 @@ def test_asset_group_sightings_jobs(flask_app, db, admin_user, test_root, reques
     uuids = [job_id2, job_id1]
 
     # Don't send anything to acm
-    with mock.patch('app.modules.asset_groups.models.current_app'):
+    with mock.patch('app.modules.asset_groups.models.current_app') as mock_app:
+        mock_app.config.get.return_value = 'zebra'
```

Comment:
This is actually the get for 'CONFIG_MODEL' Should really make sure it's just that.